nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/resources/docs/org.apache.nifi.processors.standard.Wait/additionalDetails.html

```diff
@@ -15,9 +15,9 @@
   limitations under the License.
 -->
 <head>
-    <meta charset="utf-8"/>
-    <title>ValidateCsv</title>
-    <link rel="stylesheet" href="../../../../../css/component-usage.css" type="text/css"/>
+	<meta charset="utf-8"/>
+	<title>ValidateCsv</title>
```

Comment:
Wrong copy/paste here, probably here since the processor exists...

Reply:
Thank you! Pushed a new commit with the fix.